InnerEye/ML/dataset/full_image_dataset.py

```diff
@@ -262,6 +263,39 @@ def _load_dataset_sources(self) -> Dict[str, PatientDatasetSource]:
                                     )
 
 
+def convert_channels_to_file_paths(channels: List[str],
+                                   rows: pd.DataFrame,
+                                   local_dataset_root_folder: Optional[Path],
```

Comment:
This should be Path, not Optional[Path], since the function assumes local_dataset_root_folder is always provided.